hosted-ce/etc/osg/image-config.d/30-remote-site-setup.sh

```diff
@@ -46,12 +46,9 @@ function fetch_remote_os_info {
 
 # Allow the condor user to run the WN client updater as the local users
 function setup_sudo_users {
-    users=$(get_mapped_users)
-    [[ -n $users ]] || errexit "Did not find any HTCondor-CE SCITOKENS user mappings"
-
     CONDOR_SUDO_FILE=/etc/sudoers.d/10-condor-ssh
     # Replace spaces and newlines from sort ouput with commas
-    condor_sudo_users=`tr ' \n' ',' <<< $users`
+    condor_sudo_users=`tr ' \n' ',' <<< $1`
```

Comment:
```suggestion
    condor_sudo_users=`tr ' \n' ',' <<< $users`
```

For clarity to go along with a subsequent suggestion...